tasks/eth/ink-002-set-respected-game-type/SignFromJson.s.sol

```diff
@@ -0,0 +1,49 @@
+
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";
+import {OptimismPortal2, IDisputeGame} from "@eth-optimism-bedrock/src/L1/OptimismPortal2.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {GnosisSafe} from "safe-contracts/GnosisSafe.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+
+contract SignFromJson is OriginalSignFromJson {
+    using LibString for string;
+
+    // Chains for this task.
+    address InkMainnetOptimismPortalProxy = 0x5d66C1782664115999C47c9fA5cd031f495D3e4F;
```

Comment:
We should be reading this from the registry like we do in other tasks, instead of hardcoding addresses

Reply:
got it, I'll update, I think Ink wasn't in the registry yet